src/components/programResources.scss

```diff
@@ -0,0 +1,90 @@
+.program-resources {
+  margin-bottom: 200px;
+  display: flex;
+  height: 500px;
+}
+
+.program-resources__left {
+  flex: 0 1 48%;
+}
+
+.program-resources__image {
+  background-image: url('../images/resources.jpg');
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 100%;
+  height: 100%;
+  min-height: 520px;
+}
+
+.program-resources__right {
+  flex: 0 1 50%;
+  padding-top: 48px;
+
+  h2 {
+    line-height: 59px;
+    color: var(--ocean);
+    margin-bottom: 32px;
+  }
+
+  p {
+    font-size: var(--headerFontSizeTertiary);
+    line-height: var(--headerLineHeightSecondary);
+    font-family: var(--scopeOne);
+    margin: 0;
+  }
+}
+
+.program-resources__button {
```

Comment:
Let's add `cursor: pointer;` here so it's clear that it's clickable. I'd like to add some focus/hover state to this, but we can tackle that later.